src/Sentry/Extensibility/DisabledHub.cs

```diff
@@ -21,6 +22,7 @@ public class DisabledHub : IHub, IDisposable
 
     private DisabledHub()
     {
+        Logger = new SentryStructuredLogger(this);
```

Comment:
Should this be a `NoOp` instance?

Since you don't pass `options` and all public method have a gate behind IsEnabled, it's probably fine

Reply:
I made it more explicit that we are creating a "disabled" logger here, and added a test confirming that this logger does not capture logs, even when `Options.EnableLogs = true`.